app/assets/javascripts/application.js

```diff
@@ -20,9 +18,6 @@ $(document).on('page:fetch', function() {
 $(document).on('page:change', tfm.tools.hideSpinner);
```

Comment:
This (and the above) is a turbolinks event - we should also remove anything that hooks into these events. The full list is at: https://github.com/turbolinks/turbolinks-classic#events

Reply:
I've done it for core, as there was no funcionality hook on it (only cleanups), it was easy.